tests/input/not_spacing.php

```diff
@@ -5,11 +5,11 @@
 $test = 1;
 
 if (!$test > 0) {
-    echo 1;
+    echo 1    ;
```

Comment:
This file is not appropriate place for such tests, it is supposed to test NOT spacing.

Reply:
@Majkl578 can you elaborate? I don't understand this comment

Reply:
@Ocramius I suppose that @Majkl578 is saying that we're reusing a file which was intended to test the spacing after the negation operator (`!`)

Reply:
Ah yes, that needs changing.

Reply:
I'll also rename the file than, so new contributors can identify that better as well.